test/pageobjects/UserConsent.js

```diff
@@ -0,0 +1,40 @@
+import { verifyElementCopy } from '../utils/mochaw'
+import BasePage from './BasePage.js'
+
+class UserConsent extends BasePage {
+  async consentFrameTitle() {
+    return this.$('.onfido-sdk-ui-UserConsent-consentFrame > h1')
```

Comment:
Suggest to use data attributes & avoid relying on CSS class selectors in the UI tests. Eliminates the coupling of UI tests to classnames, so they can be changed safely with any styling/layout design changes without fear of breaking the UI tests also. 

I'll raise a separate PR to update `CONTRIBUTING` doc to add this note about UI tests & CSS selectors.